api/app/src/Repository/DeputyRepository.php

```diff
@@ -44,16 +44,27 @@ public function getUidToIdMapping(): array
     public function findReportsInfoByUid(int $uid, bool $includeInactive = false): ?array
     {
         $sql = <<<SQL
-        SELECT c.firstname AS "firstName",
-               c.lastname AS "lastName",
-               c.case_number AS "caseNumber",
-               co.court_order_uid AS "courtOrderUid",
-               r.type AS "type"
-        FROM deputy d
-        LEFT JOIN client c ON c.deputy_id = d.id
-        LEFT JOIN court_order co ON co.client_id = c.id
-        LEFT JOIN report r ON r.client_id = c.id
-        LEFT JOIN court_order_deputy cod ON cod.deputy_id = d.id
+        SELECT DISTINCT
+        c.firstname AS "firstName",
+        c.lastname AS "lastName",
+        c.case_number AS "caseNumber",
+        (
+            SELECT DISTINCT string_agg(co.court_order_uid, ', ')
+            FROM court_order co
+            LEFT JOIN court_order_deputy cod ON co.id = cod.court_order_id
+            LEFT JOIN deputy d ON cod.deputy_id = d.id
+            LEFT JOIN client c ON d.id = c.deputy_id
+            LEFT JOIN report re ON c.id = re.client_id
+            WHERE d.deputy_uid = :deputyUid
+            AND r.id = re.id
+        ) AS "courtOrderUid",
+        r.type AS "type",
+        r.id
+        FROM report r
+        LEFT JOIN client c ON r.client_id = c.id
```

Comment:
If we're left joining, we could potentially return rows where the join returns a null. I think at least some of these should be inner joins (some of the left joins will have to not be nulls as they are in the WHERE condition, but we should probably use inner joins so we don't accidentally return rows with nulls by missing a condition).

I would guess it's not a problem because of the WHERE, but I can't see a reason why they can't be inner joins anyway.

Reply:
With how the data should be structured (excluding our current data issues)
Every report will be linked to a client
Every client will be linked to court order
Every court order will be linked to a deputy via the link table

I have just noticed that I need to include `AND cod.deputy_id = d.id` so we're targeting the correct deputy in the link table. So the other left joins are still required so we get the correct chain to the specific deputy

Reply:
Would it be better to enforce those assumptions with inner joins? There could potentially not be a court_order_deputy entry for a court order, and an inner join would exclude that court order.

Also, you wouldn't need the extra AND clause you mentioned if you use an inner join, because there would be no chain from court_order_deputy to the deputy table. At the moment there could be, as you'll get all the rows for the court order, regardless of deputy.

Reply:
I would expect them to be excluded on this endpoint. We're trying to retrieve reports that are linked to deputies via court orders. So there should be an entry on `court_order_deputy` 

I can't think of a situation where we don't have the chain of 
```
reports > court order > court_order_deputy > deputy
                      > client
```
If we have a deputy signed up & starting to complete reports this chain should always exist. Although looking at it again, I realised I've gone of the deputy id on the client table rather than going via court order

Reply:
I'm still not sure why we need left joins instead of inner joins, though. Perhaps we should have a chat about it tomorrow.